components/Form/TextInput/index.js

```diff
@@ -47,7 +47,7 @@ const TextInput = ({
           onChange(e);
         }}
         value={value}
-        aria-describedby={hasError ? `${name}-error` : ''}
+        aria-describedby={hasError ? `${name}-error` : name}
```

Comment:
I don't think this is correct - if there's no error you're saying that it's described by itself

Reply:
Ah, yes. This meant to be referencing the label of the text input ><